diff --git a/scripts/kmodtool b/scripts/kmodtool
--- a/scripts/kmodtool
+++ b/scripts/kmodtool
@@ -91,6 +91,9 @@
 %posttrans -n akmod-${kmodname}
 nohup ${prefix}/sbin/akmods --from-akmod-posttrans --akmod ${kmodname} &> /dev/null &
 
+%post -n akmod-${kmodname}
+[ ! -x /usr/sbin/akmods-ostree-post ] || /usr/sbin/akmods-ostree-post ${kmodname} %{_usrsrc}/akmods/%{name}-%{version}-%{release}.src.rpm
+
 %files -n akmod-${kmodname}
 %defattr(-,root,root,-)
 %{_usrsrc}/akmods/*
@@ -128,7 +131,6 @@
 {
 	if [[ "${1}" = "--custom" ]]; then
 		shift
-		local customkernel=true
 	elif [[ "${1}" = "--redhat" ]]; then
 		# this is needed for akmods
 		shift
@@ -216,7 +218,6 @@
 {
 	if [[ "${1}" = "--custom" ]]; then
 		shift
-		local customkernel=true
 	elif [[ "${1}" = "--redhat" ]]; then
 		shift
 		local redhatkernel=true
